exercises/resistor-significants/description.md

```diff
@@ -13,6 +14,6 @@ These colors are encoded as follows:
 - Grey: 8
 - White: 9
 
-Combining the numbers of the first two bands gives one its significant value. As an example, if the first band is blue (6) and the second band is grey (8), the significant value is 68. 
+Combining the numbers of the first two bands gives one the significant figures of its value in Ohm. As an example, if the first band is blue (6) and the second band is grey (8), the combined significant figures in Ohm equal the number 68. 
```

Comment:
👍 Better, thanks. I think 'in Ohm' can go too, to make it even more clear.

Reply:
Ohm should be Ohms.

Reply:
I googled that ;-) But I trust you way more than google.

Reply:
@rpottsoh should we use Ohms everywhere?

Reply:
Think of Ohms as cookies.  Conversationally you wouldn't say you have 5 cookie, but rather 5 cookies.  How many cookies in a dozen?....  Having singular Ohm in your sentence(s) may not be technically wrong, but it "sounds" odd when spoken out loud.  How many Ohms is the resistor?  It is 5 Ohms or it is 1 Ohm or it is 1/2 [an] Ohm.  Use plural when the resistor is more than 0 and <= 1.  Then there is a twist, when asking a question....  Do you have a 5 Ohm resistor I can borrow?  Ohm is singular.

Reply:
> Then there is a twist, when asking a question.... Do you have a 5 Ohm resistor I can borrow? Ohm is singular.

:+1: Like a 5-cookie jar.

Reply:
Ha, thanks for keeping the analogy going.  :smile: